package.json

```diff
@@ -21,6 +21,7 @@
     "react-native": "0.67.2",
     "react-native-config": "^1.4.5",
     "react-native-encrypted-storage": "^4.0.2",
+    "react-native-floating-action": "^1.22.0",
```

Comment:
I don't think we need extra library here. Please, take a look: https://callstack.github.io/react-native-paper/fab.html